crates/op-test-vectors/src/derivation.rs

```diff
@@ -1,8 +1,93 @@
 //! Module containing the derivation test fixture.
 
+use alloy::consensus::Blob;
+use alloy::primitives::{Bytes, B256};
 use serde::{Deserialize, Serialize};
 
 /// The derivation fixture is the top-level object that contains
 /// everything needed to run a derivation test.
-#[derive(Serialize, Deserialize, Debug, Default)]
-pub struct DerivationFixture {}
+#[derive(Serialize, Deserialize, Debug, Default, Eq, PartialEq)]
+pub struct DerivationFixture {
+    /// A list of L1 Blocks to derive from.
+    pub blocks: Vec<FixtureBlock>,
+}
```

Comment:
Oh of course, good catch. Though I think all we need here is to assert against the output root which is a hash of the state root, block hash, and message passer storage root. This effectively commits to the constructed l2 blocks since we include the block hash in the output root. (See: https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/output.go)